components/label.json

```diff
@@ -1,5 +1,7 @@
 { 
   "label": {
+    "comment":"Applied to input labels",
```

Comment:
This is not correct, it's for generic text which appears on screen.

Reply:
I didn't realise that when you made the PR. Does this feed into the localisation pipeline somehow?

Reply:
The desktop team wanted a generic component just for showing text on the screen when it isn't part of any other component, and suggested it take the name "label". So unrelated to anything in the provided link.

Also unrelated to anything localisation.

Reply:
Ok, so label is not a keyword but comment is.

Reply:
Yeah, "comment" and "figma" are special properties which appear in the input files but are ignored when processing.